lighthouse-core/gather/driver.js

```diff
@@ -912,8 +912,12 @@ class Driver {
       /**
        * @param {LH.Crdp.Security.SecurityStateChangedEvent} event
        */
-      const securityStateChangedListener = ({securityState, explanations}) => {
-        if (securityState === 'insecure') {
+      const securityStateChangedListener = ({
+        securityState,
+        explanations,
+        schemeIsCryptographic,
+      }) => {
+        if (securityState === 'insecure' && schemeIsCryptographic) {
```

Comment:
yeah, the nice thing about `schemeIsCryptographic` is that it's always testing the scheme of the URL on which this event was fired†. It would be difficult to get the current URL in here *and* (in the case of redirects) to make sure the URL we're checking is the one associated with the event we're checking.

I guess the question is are there cases where `schemeIsCryptographic` is false but we should still be checking for the `securityState`, otherwise we're testing a Chrome-generated error page?

†uh, at least I assume it is :)